spec/custy_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'custy'
+
+
+ describe "custy" do
+     
+     subject(:custy) { described_class.new(class_double) }
+     
+     
+      it "is initialised with a empty array by default" do
+          c = Custy.new
+          c.food_order
+        expect(c.food_order).to eq []
+      end
+
+    
+    
+        it "checks that subtotal is pushed to food_order" do
+            c = Custy.new
+            c.subtotal
+            expect(custy.select_food).to recieve(menu)
+        
+        end
+        
+        
+        
+        
+        
+        
+        
+    end
```

Comment:
Lint/BlockAlignment: `end` at 30, 4 is not aligned with `describe "custy" do` at 4, 1.